src/Symfony/Component/Polyfill/bootstrap.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Polyfill as p;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+
+if (PHP_VERSION_ID < 50400) {
+    define('JSON_BIGINT_AS_STRING', 2);
+    define('JSON_UNESCAPED_SLASHES', 64);
+    define('JSON_PRETTY_PRINT', 128);
+    define('JSON_UNESCAPED_UNICODE', 256);
```

Comment:
-1 for this. This breaks libraries detecting whether the feature is detected (old versions of Symfony were defining the constant at the beginning of the file, and it was changed precisely for this reason).

constants should be defined only if their behavior is also polyfilled.